### ISSUE TITLE
[Feature] Add total test time to HTML Reporter

### ISSUE THREAD

**Issue by AndyGleed** [microsoft/playwright#19556]
It would be nice if the HTML Reporter (and perhaps other reporters) could output how long the tests took to run.

**Comment by Marcin-Marczak** [comment]
Yes it is very serius problem, would be really awesome to have it implemented
